tailnet/proto/tailnet.proto

```diff
@@ -163,22 +163,23 @@ message TelemetryEvent {
 	bytes id = 1;
 	google.protobuf.Timestamp time = 2;
 	string application = 3;
-	Status status = 4;
-	string disconnection_reason = 5;
-	ClientType client_type = 6;
-	uint64 node_id_self = 7;
-	uint64 node_id_remote = 8;
-	P2PEndpoint p2p_endpoint = 9;
-	int32 home_derp = 10;
-	DERPMap derp_map = 11;
-	Netcheck latest_netcheck = 12;
-
-	google.protobuf.Duration connection_age = 13;
-	google.protobuf.Duration connection_setup = 14;
-	google.protobuf.Duration p2p_setup = 15;
-	google.protobuf.Duration derp_latency = 16;
-	google.protobuf.Duration p2p_latency = 17;
-	google.protobuf.FloatValue throughput_mbits = 18;
+	string version = 4;
```

Comment:
Since this is already used in main this needs to be a unique number we haven't used before. You can put it in whatever order you want in the proto type, but the `= 4` should be `= 19` and the other fields should not be changed